server/data/nl/danielsjf.txt

```diff
@@ -2871,3 +2871,507 @@ Onze droger is alweer stuk.
+Het schip was gezonken na een zware storm.
+Het is kwart over vier.
+Het is kwart voor drie. 
+Om half negen heb ik een afspraak bij de dokter.
+Ik vertrek om twintig over twaalf.
+Begint het al om vijf voor elf?
+Om twee over twee geven ze het startschot.
+Over drie minuten begint het.
+Nog twee uur te gaan.
+Het is vijf voor half acht.
+Het is tien over acht. 
+Nu is het achttien uur dertig.
+We hebben onze vlucht om tien over zeven 's ochtends.
+De bus vertrekt om zestien over drie stipt.
+De trein komt binnen een paar minuten aan.
+De bus komt slechts eenmaal per uur, dus maak dat je op tijd bent.
+Er is een trein om het kwartier.
+Het is belangrijk om op tijd te komen.
+Je wilt toch geen slechte indruk maken.
+Er lag een laag stof op de meubels.
+Het was duidelijk een oud huis. 
+De motten hadden gaten gemaakt in de gordijnen.
+De vloer kraakte als je erover liep.
+Als kind hadden zijn ouders hem leren schaatsen. 
+Hoeveel sprekers zou je nodig hebben om met artificiële intelligentie een goede spraakassistent te kunnen bouwen?
+Je hebt in ieder geval minstens tienduizend uur aan gesproken tekst nodig.
+Maar je kunt het dan ook gebruiken om bijvoorbeeld ondertitels aan films toe te voegen. 
+Het is wel zo dat woorden die in de data zitten beter herkent worden.
+Daarom is het noodzakelijk om zo veel mogelijk verschillende woorden toe te voegen.
+Ook vervoegingen zijn belangrijk want ik loop is een ander woord dan ik liep.
+Een huis is in het algemeen groter dan een hut.
+Zijn die twee jongens even groot?
+Jan is even groot als Joachim.
+Els is kleiner dan Elke.
+Erik is groter dan Stijn.
+Isabelle is sneller dan Jef.
+Sjoerd is sterker dan Peter.
+Marc is even goed in koken als Heidi.
+Mycroft is beter dan Alexa en Google.
+Ook Cortana is een bekende spraakassistent.
+Allemaal kunnen ze afspraken maken in je kalender.
+Verder integreren ze vaak met Spotify of andere muziekservices.
+Mijn rugzak is versleten.
+Ik moet dringend een nieuwe kopen.
+Het is toch vermoeiend om zoveel zinnen te schrijven.
+Het was een zware nacht.
+Die opmerking was sarcastisch bedoelt.
+Door de uitbarsting van de vulkaan hebben ze twee dorpen moeten evacueren.
+De evacuatie verliep niet zo vlot als ze gepland hadden.
+Hij kan veel instrumenten bespelen.
+Hij is vooral bekend van de liedjes waarin hij op een mondharmonica speelt.
+Jazz is zijn favoriete genre.
+Ga je dit jaar naar Pukkelpop?
+Nee ik ga naar Werchter.
+Ja, Tomorrowland is twee weekends.
+Na de aanvaring was hun sloep brandhout. 
+Het regent pijpenstelen.
+Waar zijn de toiletten?
+Wat is een totempaal?
+Ik heb een wigwam in mijn tuin.
+De vloer is antislip.
+Ik kan goed lassen.
+Lassie is een bekende hond.
+De renovatiewerken hebben langer geduurd dan gepland.
+Tot zover mijn nuttige bijdrage.
+Ik heb dat spel gekocht op de PlayStation.
+Mijn zus verkiest de Xbox boven de PlayStation.
+Mijn huis zit vol sensoren.
+De stoel heeft krassen gemaakt op de vloer.
+Ik vind het dekbedovertrek niet.
```

Comment:
dit is een vreemde zin: ik zou er "kan .. niet vinden" of "vond .. niet" van maken